server/src/main/java/org/elasticsearch/rest/RestRequest.java

```diff
@@ -74,8 +74,8 @@
     private HttpRequest httpRequest;
 
     private boolean contentConsumed = false;
-
     private final long requestId;
+    private final Version compatibleApiVersion;
```

Comment:
This is a nit but I prefer to keep final variables grouped together and non-final member variables grouped together. This means that these two variables should be moved under `httpChannel` and followed by an empty line, which would separate them from `httpRequest` and `contentConsumed`

Reply:
agree, I like that arrangement too